src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -93,7 +93,7 @@ function WorkspaceMembersPage(props) {
      */
     const validateSelection = useCallback(() => {
         const newErrors = {};
-        const ownerAccountID = _.first(PersonalDetailsUtils.getAccountIDsByLogins([props.policy.owner]));
+        const ownerAccountID = _.first(PersonalDetailsUtils.getAccountIDsByLogins([props.policy.owner || '']));
```

Comment:
@Pujan92 i think we should fix the issue inside `getAccountIDsByLogins()` to handle undefined values.